dv/uvm/icache/doc/ibex_icache_dv_plan.md

```diff
@@ -1,99 +1,142 @@
 ---
-title: "IBEX_ICACHE DV Plan"
+title: "Ibex ICache DV Plan"
 ---
 
 ## Goals
 * **DV**
-  * Verify all IBEX_ICACHE IP features by running dynamic simulations with a SV/UVM based testbench
-  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+  * Verify all Ibex ICache IP features by running dynamic simulations with a SV/UVM based testbench
+  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage.
 * **FPV**
-
-## Current status
-<!-- TODO for now -->
-* [Design & verification stage]({{< relref "doc/project/hw_dashboard" >}})
```

Comment:
this should link to [opentitan/doc/project/hw_dashboard](https://github.com/lowRISC/opentitan/blob/master/doc/project/hw_dashboard.md), to conform with all other DV plan documents

Reply:
This brings up another question, which I think would be more directed towards @tomroberts-lowrisc and @sjgitty - are we going to have a hw_dashboard entry for the Icache design side?

Reply:
Well, that will only work for code that has been vendored in to OpenTitan. I'm not really convinced that OpenTitan-specific integration stuff belongs here. If we eventually decide it does, let's add it back in as a separate PR.

Reply:
Yes, that is a good point, as per the comment made by @asb, we should have a later discussion to decide what should be done for status reporting and integration with other OT tooling, for the time being I think it is okay to omit it (this applies to all three of the links).